src/main/java/net/openhft/chronicle/queue/RollCycles.java

```diff
@@ -87,15 +87,15 @@ public enum RollCycles implements RollCycle {
     /**
      * 0x1fffffffff entries per day, indexing every 128th entry
      */
-    LARGE_DAILY(/*-----*/"yyyyMMdd'L'", 24 * 60 * 60 * 1000, 32 << 10, 128),
+    LARGE_DAILY(/*-----*/"yyyyMMdd'L'", 24 * 60 * 60 * 1000, MAX_INDEX_COUNT, 128),
     /**
      * 0x3ffffffffff entries per day, indexing every 256th entry
      */
-    XLARGE_DAILY(/*----*/"yyyyMMdd'X'", 24 * 60 * 60 * 1000, 128 << 10, 256),
+    XLARGE_DAILY(/*----*/"yyyyMMdd'X'", 24 * 60 * 60 * 1000, MAX_INDEX_COUNT, 256),
     /**
      * 0xffffffffffff entries per day with sparse indexing (every 1024th entry)
      */
-    HUGE_DAILY(/*------*/"yyyyMMdd'H'", 24 * 60 * 60 * 1000, 512 << 10, 1024),
+    HUGE_DAILY(/*------*/"yyyyMMdd'H'", 24 * 60 * 60 * 1000, MAX_INDEX_COUNT, 1024),
```

Comment:
Something doesn't feel right here: the max index should float along with the other parameters
(Or should even be a derivable value from the other parameters)

Reply:
@peter-lawrey this is your commit...